oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/DefaultReactiveOAuth2AuthorizedClientManager.java

```diff
@@ -167,6 +263,58 @@ public void setContextAttributesMapper(Function<OAuth2AuthorizeRequest, Mono<Map
 		this.contextAttributesMapper = contextAttributesMapper;
 	}
 
+	/**
+	 * Gets the handler that handles successful authorizations.
+	 *
+	 * @return the handler that handles successful authorizations.
+	 * @see DefaultAuthorizationSuccessHandler
+	 * @since 5.3
+	 */
+	public ReactiveOAuth2AuthorizationSuccessHandler getAuthorizationSuccessHandler() {
```

Comment:
We typically don't expose a getter for an optional collaborator - only the setter. However, we should javadoc the default collaborator.

Reply:
Removed